[codemod] markDynamoStrictTest batch 18

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #117609
* #117606
* #117605
* __->__ #117604

[codemod] markDynamoStrictTest functorch/test_vmap
[codemod] markDynamoStrictTest profiler/test_profiler_tree
[codemod] markDynamoStrictTest profiler/test_profiler
[codemod] markDynamoStrictTest profiler/test_memory_profiler
[codemod] markDynamoStrictTest functorch/test_ops
[codemod] markDynamoStrictTest functorch/test_aotdispatch

zou3519 committed:
[codemod] markDynamoStrictTest batch 18

[codemod] markDynamoStrictTest functorch/test_vmap
[codemod] markDynamoStrictTest profiler/test_profiler_tree
[codemod] markDynamoStrictTest profiler/test_profiler
[codemod] markDynamoStrictTest profiler/test_memory_profiler
[codemod] markDynamoStrictTest functorch/test_ops
[codemod] markDynamoStrictTest functorch/test_aotdispatch

[ghstack-poisoned]

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 18"


[codemod] markDynamoStrictTest functorch/test_vmap
[codemod] markDynamoStrictTest profiler/test_profiler_tree
[codemod] markDynamoStrictTest profiler/test_profiler
[codemod] markDynamoStrictTest profiler/test_memory_profiler
[codemod] markDynamoStrictTest functorch/test_ops
[codemod] markDynamoStrictTest functorch/test_aotdispatch

[ghstack-poisoned]

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 18"


[codemod] markDynamoStrictTest functorch/test_vmap
[codemod] markDynamoStrictTest profiler/test_profiler_tree
[codemod] markDynamoStrictTest profiler/test_profiler
[codemod] markDynamoStrictTest profiler/test_memory_profiler
[codemod] markDynamoStrictTest functorch/test_ops
[codemod] markDynamoStrictTest functorch/test_aotdispatch

[ghstack-poisoned]

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 18"


[codemod] markDynamoStrictTest functorch/test_vmap
[codemod] markDynamoStrictTest profiler/test_profiler_tree
[codemod] markDynamoStrictTest profiler/test_profiler
[codemod] markDynamoStrictTest profiler/test_memory_profiler
[codemod] markDynamoStrictTest functorch/test_ops
[codemod] markDynamoStrictTest functorch/test_aotdispatch

[ghstack-poisoned]

bdhirsh commented on torch/testing/_internal/dynamo_test_failures.py:
```diff
@@ -3178,6 +3172,123 @@
+    "TestProfiler.test_profiler_op_event_args",  # profiler/test_profiler
+    "TestProfiler.test_source_multithreaded_complex_work_in_main_thread_True",  # profiler/test_profiler
+    "TestProfiler.test_source_multithreaded_multiple_preexisting_work_in_main_thread_True",  # profiler/test_profiler
+    "TestAOTAutograd.test_input_mutation_aliases_and_output_alias",  # functorch/test_aotdispatch
```
Hmm does this mean that dynamo is running directly on the `test_aotdispatch.py` tests? Do we want that? (having dynamo run on the AOTAutograd code seems.. fragile)

zou3519 replied:
Answered offline, but Dynamo should graph break correctly (or handle them) when it sees things like make_fx, FunctionalTensor, FakeTensor, etc.

bdhirsh reviewed (approved):
left a q but stamping anyway

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 18"


[codemod] markDynamoStrictTest functorch/test_vmap
[codemod] markDynamoStrictTest profiler/test_profiler_tree
[codemod] markDynamoStrictTest profiler/test_profiler
[codemod] markDynamoStrictTest profiler/test_memory_profiler
[codemod] markDynamoStrictTest functorch/test_ops
[codemod] markDynamoStrictTest functorch/test_aotdispatch

[ghstack-poisoned]

zou3519 commented:
@pytorchbot revert -m "probably a crossed merge?" -c landrace

zou3519 committed:
Update on "[codemod] markDynamoStrictTest batch 18"


[codemod] markDynamoStrictTest functorch/test_vmap
[codemod] markDynamoStrictTest profiler/test_profiler_tree
[codemod] markDynamoStrictTest profiler/test_profiler
[codemod] markDynamoStrictTest profiler/test_memory_profiler
[codemod] markDynamoStrictTest functorch/test_ops
[codemod] markDynamoStrictTest functorch/test_aotdispatch

[ghstack-poisoned]

zou3519 commented:
re-landed in another PR
